### PR TITLE
Add option to add Cc recipients  

### PR DESCRIPTION
**Pull request by rdjogo**
I added the CC field because a specific use case requested it (create tasks on Asana via email, and add watchers via CC as per Asana documentation).

### CODE REVIEW

**Review by jinpingh — APPROVED** [review]
LGTM

File: actions/send_email.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 
 class SendEmail(Action):
-    def run(self, email_from, email_to, subject, message, account, mime="plain", attachments=None):
+    def run(self, email_from, email_to, subject, message, account, email_cc=[], mime="plain", attachments=None):
```
Comment (blag) [inline actions/send_email.py]: This doesn't do what you think it does, and it introduces a very frustrating subtle bug.

See this [Stack Overflow post](https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists) for more about it, and how to fix it.

**Review by blag — CHANGES_REQUESTED** [review]

File: actions/send_email.py (hunk 1 above)
Comment (rdjogo) [reply actions/send_email.py]: Thanks, will do!

Comment (rdjogo) [reply actions/send_email.py]: Better now?

**Review by blag — APPROVED** [review]
:+1:

**Comment by punkrokk** [comment]
would it make sense to add the same for BCC at this time, since that could also be a use case?

**Comment by blag** [comment]
Properly supporting BCC headers can only be done after (or as part of) fixing #25.

**Comment by rdjogo** [comment]
@blag I'll try and give #25 a go.

**Pull request merged**